### PR TITLE
hmpope-51 Applicants full name step

### PR DESCRIPTION
**Pull request by bendiggle**
Added the applicants full name step:
- Added the field to the step in routes config (field now in common),
- Moved the journey header translation to common translations so it can be used for all journeys,
- Added the acceptance steps for the step

### CODE REVIEW

File: apps/request-declaration/acceptance/features/applicants-full-name.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+});
+
+Scenario('On submitting the completed form I am taken to ref-number step', (
+  I,
+  applicantsFullNamePage,
+  refNumberPage
+) => {
+  applicantsFullNamePage.fillFormAndSubmit();
+  I.seeInCurrentUrl(refNumberPage.url);
+});
```
Comment (joefitter) [inline apps/request-declaration/acceptance/features/applicants-full-name.js:58]: We can move most of this to common we literally only want to check where the page links to in the journey (unless there are different validations or conditional translations in different journeys)

Comment (bendiggle) [reply apps/request-declaration/acceptance/features/applicants-full-name.js:58]: Oh yeah, I remember you said this yesterday! I'll update

Comment (bendiggle) [reply apps/request-declaration/acceptance/features/applicants-full-name.js:58]: @joefitter Do you think we could let this slide for now until we definitely know what we are going to do about shared steps?

Comment (joefitter) [reply apps/request-declaration/acceptance/features/applicants-full-name.js:58]: chalk it up as technical debt

**Pull request merged**